Shopify/View/Authenticate/SignUpViewController.swift

```diff
@@ -153,62 +224,23 @@ class SignUpViewController: UIViewController {
-            {
-                return true
-            }
-            return false
-        }
     
-    private func makePostRequest()
+    private func validation() ->Bool
     {
-        guard let url = URL(string: "https://48c475a06d64f3aec1289f7559115a55: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@ios-q2-new-capital-admin2-2022-2023.myshopify.com/admin/api/2023-01/customers.json") else{
-            return
-        }
-        print("Making api call")
-        
-        var request = URLRequest(url: url)
-        request.httpMethod = "POST"
-        request.httpShouldHandleCookies = false
-        request.addValue("application/json",forHTTPHeaderField: "Content-Type")
-        request.addValue("application/json",forHTTPHeaderField: "Authorization")
-       // request.addValue("application/json",forHTTPHeaderField: "Accept")
-                
-     //   request.setValue("application/json", forHTTPHeaderField: "Authorization -token")
-        print("\(emailTxt.text)")
-        let body : [String:Any] = [
-            "customer" : [
-                "email":"\(emailTxt.text!)",
-                "first_name":"\(usernameTxt.text!)",
-                "tags":"\(passwordTxt.text!)",
-                "addresses" : [["address1":"\(addressTxt.text!)"]]
-            ]
-        ]
-        request.httpBody = try? JSONSerialization.data(withJSONObject: body, options: .prettyPrinted)
-        
-        let task = URLSession.shared.dataTask(with: request) { data, _, error in
-            guard let data = data, error ==  nil else{
-                return
-            }
-            
-            do{
-                let response =  try JSONSerialization.jsonObject(with: data , options:  .allowFragments)
-                print("SUCCSESS\(response)")
-            }catch{
-                print(error)
-            }
+        if usernameTxt.text != "" && usernameTxt.text?.count ?? 0 >= 10 && usernameTxt.text?.count ?? 0 <= 20 &&
+            emailTxt.text != "" && emailTxt.text?.count ?? 0 >= 10 && emailTxt.text?.count ?? 0 <= 40 &&  passwordTxt.text != "" && passwordTxt.text?.count ?? 0 >= 10 && passwordTxt.text?.count ?? 0 <= 20 && confirmPasswordTxt.text != "" && confirmPasswordTxt.text?.count == passwordTxt.text?.count && addressTxt.text != "" && addressTxt.text?.count ?? 0 >= 5 && addressTxt.text?.count ?? 0 <= 8 &&
+            cityTxt.text != "" && cityTxt.text?.count ?? 0 >= 5 && cityTxt.text?.count ?? 0 <= 8 && countryTxt.text != "" && countryTxt.text?.count ?? 0 >= 5 && countryTxt.text?.count ?? 0 <= 8 && phoneTxt.text != "" && phoneTxt.text?.count ?? 0 >= 11 && phoneTxt.text?.count ?? 0 <= 13
+        {
```

Comment:
i don't know if it work will and you cover all edge cases , but you can search for regex and try to use it